opm/simulators/wells/SingleWellState.hpp

```diff
@@ -117,6 +118,40 @@ class SingleWellState {
       vaporized_water = 3
     };
 
+    struct GroupTarget {
+        std::string group_name;
+        Group::ProductionCMode production_cmode {Group::ProductionCMode::NONE};
+        Group::InjectionCMode  injection_cmode  {Group::InjectionCMode::NONE};
```

Comment:
Might you need one injection control mode for each phase, or at least a way to associate this "target" to a specific injection phase?

Reply:
In the current form of https://github.com/OPM/opm-simulators/pull/6545 , which focuses on production control mode mostly. The injection control mode is not of much use, while your point is valid. I will discuss it with @steink